### ISSUE TITLE
[a11y] Screen reader does not read the error description on certain types of questions

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#10436]
**Describe the bug**
The screen reader does not read the error description on the following question types:

    - Checkbox
    - Radio buttons
    - Enumerators (Also has no other indication of an error)
    - File upload (It does say “No file chosen”, but doesn’t say that there’s an error or that the field is required.)

**To Reproduce**

1. As an admin, create a program with these four question types and make the questions required.
2. As an applicant, apply to the program.
3. Turn the screen reader on.
4. In the application, press "Continue" without filling in any answers to the questions.
5. Tab to the question with an error.  
6.  Note that the screen reader never announces "Error: This question is required."

**Expected behavior**
The screen reader should read the error description, like it does for other types of questions.

**Screenshots**

**Desktop (please complete the following information):**

- OS: MacOS
- Browser: Chrome
- Screen reader:  VoiceOver
